db/migrate/20200131101835_create_chaser_emails_table.rb

```diff
@@ -0,0 +1,9 @@
+class CreateChaserEmailsTable < ActiveRecord::Migration[6.0]
+  def change
+    create_table :chaser_emails_tables do |t|
```

Comment:
Don't like `:chaser_emails_tables` much. What will the model be called? I'd probably suggest just `SentChaser` and `:sent_chasers` (to make it clear we are talking about sent notifications, not pending ones). This would allow you to write things like `application_choice.sent_chasers.any?` etc.

Reply:
Yea that was completely misnamed. It was meant to be chaser_emails, but yea sent_chaseers is way better

Reply:
Does this mean that the model will be called `ChasersSent`, not `ChaserSent`?